stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -225,6 +225,7 @@ impl DBConfig {
             StacksEpochId::Epoch22 => self.version == "3" || self.version == "4",
             StacksEpochId::Epoch23 => self.version == "3" || self.version == "4",
             StacksEpochId::Epoch24 => self.version == "3" || self.version == "4",
+            StacksEpochId::Epoch30 => todo!(), // Need input from Jude here
```

Comment:
This should be the same as 2.1-2.4 for now.

Reply:
Oki, copied